frontend/src/employee-mobile-frontend/components/attendances/notes/DailyNote.tsx

```diff
@@ -65,6 +66,12 @@ export default React.memo(function DailyNote({ child }: Props) {
               </span>
             </FixedSpaceColumn>
           )}
+          {groupNote && (
+            <FixedSpaceColumn spacing={'xxs'}>
+              <Label>{i18n.attendances.notes.labels.groupNotesHeader}</Label>
+              <span>{groupNote.note}</span>
+            </FixedSpaceColumn>
+          )}
         </FixedSpaceColumn>
       ) : (
         <div>{i18n.attendances.notes.noNotes}</div>
```

Comment:
If the child does not have an daily note the groupNote will not be shown. How about adding the groupNote in different spot?

```suggestion
</FixedSpaceColumn>
      ) : (
        <>
          {groupNote ? (
            <FixedSpaceColumn spacing={'xxs'}>
              <Label>{i18n.attendances.notes.labels.groupNotesHeader}</Label>
              <span>{groupNote.note}</span>
            </FixedSpaceColumn>
          ) : (
            <div>{i18n.attendances.notes.noNotes}</div>
          )}
        </>
```